src/site/layouts/campaign_landing_page.drupal.liquid

```diff
@@ -117,67 +117,72 @@
       </div>
 
       <!-- Spotlight -->
-      <div class="spotlight">
-        <!-- Title -->
-        <p class="grey-header">SPOTLIGHT</p>
-        <h2>{{ fieldClpSpotlightHeader }}</h2>
-        <p>
-          {{ fieldClpSpotlightIntroText }}
-          <a href="{{ fieldClpSpotlightCta.entity.fieldButtonLink.url }}" rel="noreferrer noopener" target="_ blank">
-            {{ fieldClpSpotlightCta.entity.fieldButtonLabel }}
-          </a>
-        </p>
-
-        <!-- List of spotlights -->
-        {% if fieldClpSpotlightLinkTeasers != empty %}
-          <div class="link-teasers">
-            {% for linkTeaser in fieldClpSpotlightLinkTeasers %}
-              <div class="link-teaser">
-                <a href="{{ linkTeaser.entity.fieldLink.url }}" rel="noreferrer noopener" target="_blank">{{ linkTeaser.entity.fieldLink.title }}</a>
-                <p>{{ linkTeaser.entity.fieldLinkSummary }}</p>
-              </div>
-            {% endfor %}
-          </div>
-        {% endif %}
+      <div class="spotlight usa-grid usa-grid-full vads-u-padding-y--4 vads-u-padding-x--2 medium-screen:vads-u-padding-x--0">
+        <div class="usa-width-three-fourths">
+          <!-- Title -->
+          <p class="grey-header vads-u-font-size--sm vads-u-margin-bottom--0">SPOTLIGHT</p>
+          <h2 class="vads-u-margin-top--0">{{ fieldClpSpotlightHeader }}</h2>
+          <p class="vads-u-margin-top--1 vads-u-margin-bottom--2">
+            {{ fieldClpSpotlightIntroText }}
+            <a href="{{ fieldClpSpotlightCta.entity.fieldButtonLink.url }}" rel="noreferrer noopener" target="_ blank">
+              {{ fieldClpSpotlightCta.entity.fieldButtonLabel }}
+            </a>
+          </p>
+
+          <!-- List of spotlights -->
+          {% if fieldClpSpotlightLinkTeasers != empty %}
+            <div class="link-teasers vads-u-display--flex vads-u-flex-wrap--wrap">
+              {% for linkTeaser in fieldClpSpotlightLinkTeasers %}
+                <div class="link-teaser medium-screen:vads-l-col--4 medium-screen:vads-u-margin-right--2 vads-u-margin-top--2 vads-u-display--flex vads-u-flex-direction--column vads-u-background-color--gray-lightest vads-u-padding--2">
+                  <a class="vads-u-font-weight--bold" href="{{ linkTeaser.entity.fieldLink.url }}" rel="noreferrer noopener" target="_blank">{{ linkTeaser.entity.fieldLink.title }}</a>
```

Comment:
I don't think we need all these links to have the rel or target attributes because I don't think they'll go to external websites. Or maybe they will?

Reply:
If they do end up all opening in a new window, we'll need to announce that to a screen reader. An easy way about be something like: `aria-label="{{ linkTeaser.entity.fieldLink.title }} (opens in new window)"`